kentik_api_library/tests/component/test_alerting.py

```diff
@@ -0,0 +1,34 @@
+from http import HTTPStatus
+
+from kentik_api.api_resources.alerting_api import AlertingAPI
+from kentik_api.api_calls.api_call import APICallMethods
+from kentik_api.public.manual_mitigation import ManualMitigation
+from tests.component.stub_api_connector import StubAPIConnector
+
+
+def test_create_manual_mitigation_success() -> None:
```

Comment:
Isn't that just a unit test of `alerting_api.create_manual_mitigation`? I see that it is in `component` directory. Could you explain why not in `unit`?

Reply:
The reason why this test is in `component` directory is analogy with other modules tests which are also placed in `component` directory. After some considerations I am not sure if we should keep `component` directory. Maybe better idea is to move all tests from `component` to `unit`?